enforcer/lookup/lookup.go

```diff
@@ -29,14 +32,38 @@ func NewPolicyDB() (m *PolicyDB) {
 	m = &PolicyDB{
 		numberOfPolicies: 0,
 		equalMapTable:    map[string]map[string][]*ForwardingPolicy{},
+		equalPrefixes:    map[string][]int{},
```

Comment:
New type for the []int{} will be better such as IntList. Which has an interface with a method called SortedInsert to start with.